packages/react/src/__tests__/ReactProfilerDOM-test.internal.js

```diff
@@ -140,7 +137,7 @@ describe('ProfilerDOM', () => {
 
               // Evaluate in an unwrapped callback,
               // Because trace/wrap won't decrement the count within the wrapped callback.
-              setImmediate(() => {
+              Promise.resolve().then(() => {
```

Comment:
I kind of wish this could be re-used, but it's unclear where something like that could go. .

Reply:
Could do an `await`, then you're guaranteed a tick even if promises are faked

Reply:
Naw, I think this is fine.